frontend/sw360-portlet/src/main/resources/META-INF/resources/html/search/view.jsp

```diff
@@ -102,6 +106,18 @@
 
 		</div>
 	</div>
+    <div class="row">
+        <h4><liferay-ui:message key="note.if.unchecked.entire.document.then.search.will.be.restricted.to" /></h4>
+    </div>
+    <div class="row restricted-search-note">
+        <h4><liferay-ui:message key="name.for.project.component.and.release" /></h4>
+    </div>
+    <div class="row restricted-search-note">
+        <h4><liferay-ui:message key="fullname.for.license.user.and.vendor" /></h4>
+    </div>
+    <div class="row restricted-search-note">
+        <h4><liferay-ui:message key="title.for.obligation" /></h4>
+    </div>
```

Comment:
This is how it looks now:
![image](https://user-images.githubusercontent.com/50870174/216539266-b6d83e65-9d74-4431-9ac1-a3a1fdf23771.png)

I would suggest to make use of bootstrap alert css and change it to something like this using the code below:
![image](https://user-images.githubusercontent.com/50870174/216539456-211c6a85-4d8e-40aa-82fd-38b681e668f8.png)

```
<div class="alert alert-warning mb-0 w-50 alert-dismissible">
<button type="button" class="close" data-dismiss="alert">×</button>
Note: If <strong>Entire Document</strong> checkbox is unchecked, then search will be restricted to:
    <ul class="mb-0">
        <li>
            <strong>Name</strong> for <u>Project</u>, <u>Component</u> and <u>Release</u>
        </li>
        <li>
            <strong>Fullname</strong> for <u>License</u>, <u>User</u> and <u>Vendor</u>
        </li>
        <li>
            <strong>Title</strong> for  <u>Obligation</u>
        </li>
    </ul>
</div>
```